uk_results/migrations/0030_populate_postresult_post_election.py

```diff
@@ -0,0 +1,88 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+from collections import defaultdict
+
+from django.db import migrations, models
+
+
+def set_post_election_from_post(apps, schema_editor):
+    """
+    This is far from idea. Try to guess the PostExtraElection
+    that this PostResult relates to. This will have to be done by looking
+    and the related memberships and assuming they're correct (sometimes they
+    won't be, and that will have to be fixed manually later).
```

Comment:
Could you clarify why they might not be? I can think of several possibilities - but, for example, if it's the case that when `CandidateResult`s are created the `Membership` they're associated with might randomly be for the wrong election, that sounds very hard to fix even manually, and might need a different approach (e.g. comparing the data from a dump from before Thursday to the current data.)